code-of-conduct-languages/ko.md

```diff
@@ -5,24 +5,24 @@ CNCF 커뮤니티 행동 강령(Code of Conduct) v1.0
 
 ### 기여자 행동 강령
 
-본 프로젝트의 기여자 및 메인테이너(maintainer)로서 개방적이고 친근한 분위기의 커뮤니티 조성을 위하여, 이슈 보고, 기능 요청, 문서 업데이트, 풀 리퀘스트(pull request) 또는 패치 제출, 그리고 기타 다른 활동으로 기여하는 모든 분들을 존중할 것을 약속합니다.
+본 프로젝트의 기여자(contributor) 및 메인테이너(maintainer)로서 개방적이고 친근한 분위기의 커뮤니티 조성을 위하여, 이슈 보고, 기능 요청, 문서 업데이트, 풀 리퀘스트(pull request) 또는 패치(patch) 제출, 그리고 기타 다른 활동으로 기여하는 모든 분들을 존중할 것을 서약합니다.
 
-우리는 경험의 수준, 성별, 성 정체성 및 표현(gender identity and expression), 성적 지향, 장애, 외양, 신체 크기, 인종, 민족, 나이, 종교, 또는 국적에 상관 없이 모두가 차별 없는 환경에서 본 프로젝트에 참여할 수 있도록 최선을 다하고 있습니다.
+우리는 경험의 수준, 성별, 성 정체성 및 표현(gender identity and expression), 성적 지향, 장애, 외양, 신체 크기, 인종, 민족, 연령, 종교, 또는 국적에 상관없이 모두가 차별 없는 환경에서 본 프로젝트에 참여할 수 있도록 최선을 다할 것입니다.
 
 참여자에게 금지하는 행위의 예시는 다음과 같습니다.
 
 -	성적인 언어 또는 이미지 사용
--	인신 공격
+-	인신공격
 -	도발적이거나 모욕/경멸적인 코멘트
 -	공개적이거나 사적인 괴롭힘
--	타인의 주소 및 전자주소와 같은 개인 정보의 동의 없는 공개
+-	타인의 주소 및 전자 주소와 같은 개인 정보의 동의 없는 공개
 -	기타 비윤리적이거나 비전문적인 행동
 
-프로젝트 메인테이너에게는 본 행동 강령을 위반하는 코멘트, 커밋(commit), 코드, 위키(wiki) 수정, 이슈, 그리고 그 밖의 기여에 대해서 삭제, 수정, 거부할 수 있는 권한과 책임이 있습니다. 프로젝트 메인테이너는 프로젝트 관리의 모든 관점에서 이러한 행동 강령 원칙을 공정하고 일관되게 적용할 것을 약속해야 합니다. 행동 강령을 준수하지 않거나 시행하지 않는 프로젝트 메인테이너는 프로젝트 팀에서 영구적으로 제적될 수 있습니다.
+프로젝트 메인테이너에게는 본 행동 강령을 위반하는 코멘트, 커밋(commit), 코드, 위키(wiki) 수정, 이슈, 그리고 그 밖의 기여에 대해서 삭제, 수정, 거부할 수 있는 권한과 책임이 있습니다. 프로젝트 메인테이너는 프로젝트 관리의 모든 관점에서 이러한 행동 강령 원칙을 공정하고 일관되게 적용할 것을 약속해야 합니다. 행동 강령을 준수하지 않거나 시행하지 않는 프로젝트 메인테이너는 프로젝트팀에서 영구적으로 제명될 수 있습니다.
 
-본 행동 강령은 프로젝트 활동 영역 내에서 뿐만 아니라 개인이 프로젝트 또는 커뮤니티를 대변하는 공공의 활동 영역에서도 적용됩니다.
+본 행동 강령은 프로젝트 활동 영역 내에서뿐만 아니라 개인이 프로젝트 또는 커뮤니티를 대변하는 경우 공공의 활동 영역에서도 적용됩니다.
 
-쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위는 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
+쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위를 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
```

Comment:
A nit, @kriskim8937 I prefer the existing expression considering the original English expression is `behavior in Kubernetes may be reported`.

```suggestion
쿠버네티스(Kubernetes)에서의 폭력, 학대 또는 기타 허용되지 않는 행위는 [쿠버네티스 행동 강령 위원회](https://git.k8s.io/community/committee-code-of-conduct)에 이메일 <conduct@kubernetes.io>를 통해 신고할 수 있습니다. 다른 프로젝트의 경우는 CNCF 프로젝트 메인테이너 또는 중재자인 Mishi Choudhary의 이메일 <mishi@linux.com>으로 문의해 주시기 바랍니다.
```

Reply:
I agree, thank you for the suggestion.